src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkTask.java

```diff
@@ -71,9 +71,11 @@ public void put(Collection<SinkRecord> records) {
       log.warn(
           "Write of {} records failed, remainingRetries={}",
           records.size(),
-          remainingRetries,
-          sqle
```

Comment:
BTW: this will log the first exception in sqle, which is now covered in the following foreach loop. So I removed it here to prevent duplicate log.

Reply:
This will prevent the log message from containing the stack trace for `sqle`. It'd be better to include that here and to exclude it from the list below.

Reply:
Yeah I'll add it back.